recipe/build.sh

```diff
@@ -7,7 +7,6 @@ mkdir -p ${PREFIX}/lib
 [[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/nvrtc*.pc
 
 [[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
-[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
```

Comment:
you don't really need this change - then I think you just have a clean pull from conda-forge's feedstock 🎉